access.go

```diff
@@ -648,23 +660,17 @@ func (s *Server) FinishAccessRequest(w *Response, r *http.Request, ar *AccessReq
 		w.Output["expires_in"] = ret.ExpiresIn
 		if ret.RefreshToken != "" {
 			w.Output["refresh_token"] = ret.RefreshToken
-
-			refreshTokenJwt := decodeToken(ret.RefreshToken)
-			if refreshTokenJwt != nil {
-				w.Output["refresh_expires_in"] = refreshTokenJwt.Expiration - refreshTokenJwt.IssueAt
-				if !ar.SkipSetCookie {
-					AddTokenInCookie(w, ret.RefreshToken, "refresh_token", refreshTokenJwt.Expiration)
-				}
+			w.Output["refresh_expires_in"] = ret.RefreshExpireIn
+			if !ar.SkipSetCookie {
+				AddTokenInCookie(w, ret.RefreshToken, "refresh_token", int64(int32(time.Now().Unix())+ret.RefreshExpireIn))
```

Comment:
@arya-ab about my comment in this link https://github.com/AccelByte/osin/pull/27#issuecomment-1250472771

for **Use Case 2**, where we will specify the `ret.RefreshExpireIn` value that need to use the previous extended refresh token expiration value?
Is it in consumer (IAM) generator.go

Reply:
@marselsampe yes, we can specify it in generator.go. e.g: 
![image](https://user-images.githubusercontent.com/83328142/191394366-97ac1ea9-02e7-43e8-8f6b-26f3249409a3.png)

Reply:
@arya-ab I see, but that means this refresh_token cookie will not use the expiration value from that generator.go
since we call generator.go in the middle of this function (screenshot below), and `ret.RefreshExpireIn` will stay as is.
CMIIW

![image](https://user-images.githubusercontent.com/17809418/191398247-fc262c2c-ebf9-49f2-becf-21c2558f0435.png)

Reply:
yes.. we need make sure logic to generated token in generator.go contains the same expiration time with ret.RefreshExpireIn @marselsampe

Reply:
in order to make refresh_token cookie contains the same expiration time with refresh_expire_in from access.go we can modify generator.go like this https://bitbucket.org/accelbyte/justice-iam-service/commits/8efd3f788f0b3e340f90b3fba41efbeadb17a184 @marselsampe